services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/DigitalGreenCertificateToProtobufMapping.java

```diff
@@ -19,23 +22,42 @@
 @Component
 public class DigitalGreenCertificateToProtobufMapping {
 
+  static final String TEST_RESULT_ID = "covid-19-lab-result";
+  static final String TEST_MANF_ID = "covid-19-lab-test-manufacturer-and-name";
+  static final String TEST_TYPE_ID = "covid-19-lab-test-type";
+  static final String DISEASE_AGENT_TARGETED_ID = "disease-agent-targeted";
+  static final String VACCINE_PROPHYLAXIS_ID = "sct-vaccines-covid-19";
+  static final String VACCINE_MAH_ID = "vaccines-covid-19-auth-holders";
+  static final String VACCINE_MEDICINAL_PRODUCT_ID = "vaccines-covid-19-names";
```

Comment:
why is it called `_ID`?

Reply:
That's the strange mechanism they came up with: You get the valueSet `metadata`, where you exchange the fixed `ID` for the `hash`. Then, using the `hash`, you can get the actual valueSet...